app/views/conversations/recent.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Recent Conversations</h1>
+<ul class="conversation-list">
+<% @recent_conversations.each do |conversation| %>
+    <li><%= link_to conversation.name, conversation_path(conversation) %> - last comment on <%= conversation.messages.first.created_at %></li>
```

Comment:
Any time you're chaining method calls and breaking the Law of Demeter by reaching through objects to other objects, think about putting a method on the object that you have direct access to, e.g. `conversation.most_recent_message_time`.